tests/settings.py

```diff
@@ -102,6 +102,12 @@
 if 'RESOLWE_DOCKER_COMMAND' in os.environ:
     FLOW_DOCKER_COMMAND = os.environ['RESOLWE_DOCKER_COMMAND']
 
+# Ignore errors in tests when pulling Docker images from 'list_docker_images --pull' command
+FLOW_DOCKER_IGNORE_PULL_ERRORS = strtobool(os.environ.get('RESOLWE_DOCKER_IGNORE_PULL_ERRORS', '1'))
+
+# Don't pull Docker images (default is to pull, but we can disable this on Vagrant boxes)
```

Comment:
I would change this to something like:
```
# Don't pull Docker images if set via environment variable.
```
These are actually test project's settings, so there is no need to document the function in the Vagrant context. Unfortunately, we still [lack documentation of all available Resolwe's settings](https://github.com/genialis/resolwe/issues/189).